### PR TITLE
Inb4 oranges says SWAT has no place in this game

### PR DESCRIPTION
**Pull request by Bawhoppen**
:cl:
add: You can now order SWAT vest alternatively to SWAT suits at cargo for the same price.
tweak: SWAT armor is now better than sec armor again.
tweak: Simple Mob NPCs that have SWAT helms now have SWAT vests to go with instead of regular armor.
/:cl:

Same PR as https://github.com/tgstation/tgstation/pull/19760 however cargo prices have been fixed so there shouldn't be a problem

### CODE REVIEW

**Comment by optimumtact** [comment]
swat has no place in this game and op can't inb4

**Comment by PKPenguin321** [comment]
>opened with conflicts

**Comment by Bawhoppen** [comment]
its a recreation of old pr onesec

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -102,7 +102,7 @@
 	desc = "An extremely robust, space-worthy helmet in a nefarious red and black stripe pattern."
 	icon_state = "swatsyndie"
 	item_state = "swatsyndie"
-	armor = list(melee = 40, bullet = 30, laser = 30,energy = 30, bomb = 50, bio = 90, rad = 20)
+	armor = list(melee = 55, bullet = 40, laser = 35,energy = 40, bomb = 55, bio = 90, rad = 20)
```
Comment (optimumtact) [inline code/modules/clothing/head/helmet.dm]: no

File: code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -400,4 +400,9 @@
 	icon_state = "knight_templar"
 	item_state = "knight_templar"
 
-
+/obj/item/clothing/suit/armor/swat
+	name = "\improper SWAT vest"
+	desc = "A lightweight assault SWAT vest."
+	icon_state = "swatvest"
+	armor = list(melee = 55, bullet = 40, laser = 35,energy = 40, bomb = 55, bio = 90, rad = 20)
```
Comment (optimumtact) [inline code/modules/clothing/suits/armor.dm]: no

File: code/modules/clothing/head/helmet.dm (hunk 1 above)
Comment (Bawhoppen) [inline code/modules/clothing/head/helmet.dm]: we talked about this, steelpoints buff was retarded, 
If I make a PR to lower that will you megre it?

Comment (Bawhoppen) [inline code/modules/clothing/head/helmet.dm]: as in regular sec armor

**Comment by Cheridan** [comment]
no

**Comment by Bawhoppen** [comment]
cheridan come on work with me

**Comment by jughu** [comment]
Yes

**Comment by PKPenguin321** [comment]
swat armor is stupid, why does it even exist for cargo

**Comment by KorPhaeron** [comment]
conflicts and the head coder said no

**Comment by Thunder12345** [comment]
SWAT has no place in this game

**Comment by Bawhoppen** [comment]
i've been busy lately, reopen this and I will work on it later

**Comment by optimumtact** [comment]
You are literally delusional, you have made the same pr at least three times now and every single time had it denied and you keep making it.

I wish @Cheridan would github ban you so we dont' have to put up with your nonsense any longer

**Comment by Bawhoppen** [comment]
show me the 3 times i have done the exact same thing

**Comment by Bawhoppen** [comment]
okay I submit reopen and I will remove swat fromcargo

**Pull request closed without merging**